src/ocm/components/featureSupportLevels/utils.tsx

```diff
@@ -53,3 +60,87 @@ export const getLimitedFeatureSupportLevels = (
     return undefined;
   }
 };
+
+const isArmSupported = (version: string, versions: OpenshiftVersionOptionType[]) => {
+  const versionOption = versions.find((option) => option.value === version);
+  return !!versionOption?.cpuArchitectures?.includes(CpuArchitecture.ARM);
+};
+const clusterExistsReason = 'This option is not editable after the draft cluster is created';
+
+const getSNODisabledReason = (cluster: Cluster | undefined, isSupported: boolean) => {
+  if (cluster) {
+    return clusterExistsReason;
+  }
+  if (!isSupported) {
+    return 'Single-Node OpenShift is not supported in this OpenShift version';
+  }
+  return '';
+};
+
+const getArmDisabledReason = (
+  cluster: Cluster | undefined,
+  version: string,
+  versions: OpenshiftVersionOptionType[],
```

Comment:
What is the difference between `version` and `versions`?
Can we give `version` a name that aligns with its context?